gulpfile.js

```diff
@@ -208,6 +209,47 @@ gulp.task('manifest:testing', function () {
 
   // Exclude chromereload script in production:
     .pipe(jsoneditor(function (json) {
+      json.permissions = [...json.permissions, 'webRequestBlocking', 'http://localhost/8889']
+      return json
+    }))
+
+    .pipe(gulp.dest('./dist/', { overwrite: true }))
+})
+
+const scriptsToExcludeFromBackgroundDevBuild = {
+  'bg-libs.js': true,
+}
+
+gulp.task('manifest:testing-local', function () {
+  return gulp.src([
+    './dist/firefox/manifest.json',
+    './dist/chrome/manifest.json',
+  ], {base: './dist/'})
+
+    .pipe(jsoneditor(function (json) {
+      json.background = {
+        ...json.background,
+        scripts: json.background.scripts.filter(scriptName => !scriptsToExcludeFromBackgroundDevBuild[scriptName]),
+      }
+      json.permissions = [...json.permissions, 'webRequestBlocking', 'http://localhost/8889']
```

Comment:
```suggestion
      json.permissions = [...json.permissions, 'webRequestBlocking', 'http://localhost:8889']
```
I'm guessing that `8889` is a port, not a path?

Reply:
So, I don't understand why, but `http://localhost/8889` and `http://localhost/*` work, but `http://localhost:8889`, `http://localhost:*` and `http://localhost` do not

Reply:
Interesting - apparently the match pattern here doesn't support ports?
https://bugzilla.mozilla.org/show_bug.cgi?id=1362809

I think we're better off omitting the port rather than including this non-existent "8999" path. I'm also curious whether this works on Firefox

Reply:
Thoughts on whether we should change this to `localhost/*`? I remain confused about why `localhost/8999` works in the first place. If you feel inclined to leave it as `localhost/8999` we should at least make sure it works on Firefox, as I can't see any documentation to explain why it _does_ work, so it could be some sort of fluke.